packages/zero/package.json

```diff
@@ -113,6 +113,14 @@
       "types": "./out/zero-server/src/mod.d.ts",
       "default": "./out/zero/src/server.js"
     },
+    "./server/adapters/postgresjs": {
+      "types": "./out/zero-server/src/adapters/postgres.d.ts",
+      "default": "./out/zero-server/src/adapters/postgresjs.js"
+    },
+    "./server/adapters/drizzle": {
```

Comment:
I think there should be multiple exports here - separating out `node-postgres`, drizzle `node-postgres`, plain `postgres.js`, drizzle `postgres.js`. This would map better onto Drizzle adapters since the underlying client/types are different between those drizzle adapters.

They also have `neon-postgres` and `vercel-serverless`, which users may want as well.

https://github.com/drizzle-team/drizzle-orm/tree/main/drizzle-orm/src